RadixWallet/MIGRATE_TO_SARGON/Stage2/Profile/Profile/Profile+AuthorizedDapps+Edit.swift

```diff
@@ -142,7 +142,7 @@ extension Profile {
 		let dapp = try validateAuthorizedPersonas(of: unvalidatedAuthorizedDapp)
 		let networkID = dapp.networkID
 		let network = try network(id: networkID)
-		if network.authorizedDapps.contains(dapp) {
```

Comment:
Bad check, this will do a hash comparison, and when a new persona is choosed for the login, then the hashes will be different, instead the stable dapp def address is checked.